src/components/RadioGroup/RadioGroup.props.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { RadioChangeEvent } from 'antd'
+
+export type RadioGroupOption = {
+  value: string | number | boolean;
```

Comment:
would it make sense to use a generic T to allow any kind of value to be supplied?

Reply:
I changed in order to have a generic value type